Tests/CorePromise/WhenTests.swift

```diff
@@ -241,7 +242,8 @@ class WhenTests: XCTestCase {
         let p2 = Promise<Void>(error: Error.test2)
         let p3 = Promise<Void>(error: Error.test3)
 
-        when(fulfilled: p1, p2, p3).catch { error in
+        let whenFulfilledP1P2P3: Promise<Void> = when(fulfilled: p1, p2, p3)
```

Comment:
There can be also `let whenFulfilledP1P2P3: Promise<(Void, Void, Void)> = when(fulfilled: p1, p2, p3)`.